common/utils.js

```diff
@@ -36,4 +36,22 @@ export function calcOrderTotal(cart, speakers) {
     }
 
     return roundCurrency(orderTotal);
-}
\ No newline at end of file
+}
+
+const seedProducts = () => {
+    const alreadyProducts = localStorage.getItem(seedProducts);
+
+    if (!alreadyProducts) {
+        const stringSpeakers = JSON.stringify(speakers);
+        localStorage.setItem(seedProducts, stringSpeakers);
+    }
+};
+
+export const getProducts = () => {
+    seedProducts();
+
+    const products = localStorage.getItem(seedProducts);
```

Comment:
It looks like you're calling the function `seedProducts` in the localStorage object as opposed to setting it with a string `'seedProducts'`.